{"cage0x0002": {"error": "HTTPConnectionPool(host='cage0x0002', port=8080): Max retries exceeded with url: /BoardData (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C211948390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))"}, "cage0x0005": {"error": "HTTPConnectionPool(host='cage0x0005', port=8080): Max retries exceeded with url: /BoardData (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C211982690>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))"}, "cage0x0003": {"error": "HTTPConnectionPool(host='cage0x0003', port=8080): Max retries exceeded with url: /BoardData (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C2119A5990>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))"}, "cage0x0011": {"error": "HTTPConnectionPool(host='cage0x0011', port=8080): Max retries exceeded with url: /BoardData (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C211938790>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))"}, "cage0x0006": {"error": "HTTPConnectionPool(host='cage0x0006', port=8080): Max retries exceeded with url: /BoardData (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x000001C21197F5D0>: Failed to resolve 'cage0x0006' ([Errno 11001] getaddrinfo failed)\"))"}, "cage0x0004": {"error": "HTTPConnectionPool(host='cage0x0004', port=8080): Max retries exceeded with url: /BoardData (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x000001C2119698D0>: Failed to resolve 'cage0x0004' ([Errno 11001] getaddrinfo failed)\"))"}, "cage0x0007": {"sensors_values": "(255, 255, 181, 210)", "star_wheel_status": "overload", "unloader_status": "normal", "mode": "idle"}, "cage0x0008": {"error": "HTTPConnectionPool(host='cage0x0008', port=8080): Max retries exceeded with url: /BoardData (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x000001C211975FD0>: Failed to resolve 'cage0x0008' ([Errno 11001] getaddrinfo failed)\"))"}, "cage0x0015": {"error": "HTTPConnectionPool(host='cage0x0015', port=8080): Max retries exceeded with url: /BoardData (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C2119C0BD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))"}, "cage0x0010": {"error": "HTTPConnectionPool(host='cage0x0010', port=8080): Max retries exceeded with url: /BoardData (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x000001C211992D50>: Failed to resolve 'cage0x0010' ([Errno 11001] getaddrinfo failed)\"))"}, "cage0x0009": {"sensors_values": "(255, 255, 1, 222)", "star_wheel_status": "normal", "unloader_status": "normal", "mode": "pnp"}, "cage0x0012": {"error": "HTTPConnectionPool(host='cage0x0012', port=8080): Max retries exceeded with url: /BoardData (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x000001C2119B3B50>: Failed to resolve 'cage0x0012' ([Errno 11001] getaddrinfo failed)\"))"}, "cage0x0013": {"sensors_values": "(255, 1, 1, 198)", "star_wheel_status": "normal", "unloader_status": "normal", "mode": "pnp"}, "cage0x0014": {"error": "HTTPConnectionPool(host='cage0x0014', port=8080): Max retries exceeded with url: /BoardData (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x000001C2119C3010>: Failed to resolve 'cage0x0014' ([Errno 11001] getaddrinfo failed)\"))"}}